llvm/include/llvm/IR/IRBuilder.h

```diff
@@ -1992,8 +1992,9 @@ class IRBuilderBase {
   /// block.
   Constant *CreateGlobalStringPtr(StringRef Str, const Twine &Name = "",
                                   unsigned AddressSpace = 0,
-                                  Module *M = nullptr) {
-    GlobalVariable *GV = CreateGlobalString(Str, Name, AddressSpace, M);
+                                  Module *M = nullptr, bool AddNull = true) {
```

Comment:
As a side note, CreateGlobalStringPtr should be removed in favor of CreateGlobalString, as they are identical since the opaque pointer migration.